### PR TITLE
Add two numbers in a Linked list in C++ 

### PR DESCRIPTION
**Pull request by pranjwalsingh236**
…option section in C-plus-plus

### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

YES

### Description

Added the required solution to the problem 'Add two numbers' in C++ with detailed explanation

### Checklist

- [ yes] I've read the contribution guidelines.
- [ yes] I've checked the issue list before deciding what to submit.
- [ yes] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

Thanks for assigning issue 2897 to me

### CODE REVIEW

**Review by HarshCasper — CHANGES_REQUESTED** [review]
- Use Comments above the Line of Code
- Format the Code using a Clang Formatter
- Change the name of file. It looks like as if you are adding two numbers

File: C-Plus-Plus/other/addTwoNumbers.cpp [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+Enter the elements of linked list 2: 
+0
+The resulting sum of the two lists is: 
+0
+
+
+SPACE COMPLEXITY = O(max(m,n))(where m and n are the sizes of the list1 and list2 respectively)
+TIME COMPLEXITY = O(max(m,n))
+
+*/
```
Comment (bhav09) [inline C-Plus-Plus/other/addTwoNumbers.cpp]: You need to add a line at EOF. Rest everything is good.

**Review by bhav09 — CHANGES_REQUESTED** [review]
Why are you merging a file twice? Please have a look at it. 
Reviewed.

File: C-Plus-Plus/other/addTwoNumbers.cpp (hunk 1 above)
Comment (pranjwalsingh236) [reply C-Plus-Plus/other/addTwoNumbers.cpp:160]: Can u tell what is an EOF

**Review by bhav09 — COMMENTED** [review]
It is better, now.

**Review by HarshCasper — APPROVED** [review]

**Pull request merged**